evalai/get_token.py

```diff
@@ -5,6 +5,7 @@
 
 from evalai.utils.config import AUTH_TOKEN_PATH
 import json
+from json.decoder import JSONDecodeError
```

Comment:
@aaaakshat can you reorder the imports.

Can you move `import json` to line 2

Reply:
Hey @Ram81 I've done so on my latest commit.